docs/runwhen/contrib.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to CodeCollections/CodeBundles
+
+## Creating a New CodeCollection
```

Comment:
can we add the link to official doc here? 

https://docs.runwhen.com/public/v/runwhen-authors/codecollection-development/getting-started/running-your-first-codebundle

Reply:
Not sure about `Writing a Non-trivial CodeBundle`